### PR TITLE
Create Al-Madina.com.xml

### PR DESCRIPTION
**Pull request by Giltyhub**

### CODE REVIEW

File: src/chrome/content/rules/Al-Madina.com.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--
+	Secure Connection Failed:
+		madina.al-madina.com
+		waseet.al-madina.com
```
Comment (YegorIevlev) [inline src/chrome/content/rules/Al-Madina.com.xml]: Mismatched, CN=*.prod.ams3.secureserver.net.

File: src/chrome/content/rules/Al-Madina.com.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!--
+	Mismatched:
+		madina.al-madina.com, CN=*.prod.ams3.secureserver.net.
+	Secure Connection Failed:
+		waseet.al-madina.com
```
Comment (YegorIevlev) [inline src/chrome/content/rules/Al-Madina.com.xml]: Same with this one.

Comment (Giltyhub) [reply src/chrome/content/rules/Al-Madina.com.xml]: strange got a secure connection failed yesterday :thinking:

**Comment by Giltyhub** [comment]
though it doesn't impact functionality, added platform="mixedcontent" nevertheless

**Pull request merged**